compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_type_gen.bal

```diff
@@ -516,7 +503,7 @@ function createObjectAttachedFunction(jvm:MethodVisitor mv, bir:BAttachedFunctio
     // Load flags
     // TODO: get the flags
     int visibility = 1;
-    if (attachedFunc.visibility == "PACKAGE_PRIVATE") {
+    if (attachedFunc.flags == 0) {
```

Comment:
This need to be changed like (mask & flag) == flag

Reply:
Fixed